docs/References/elementTemplateSupport.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 2
+---
+import styles from './styling/elementTemplateSupport.css';
+import fileAssociations from './img/fileAssociations.png';
+import singleSchema from './img/singleSchema.png';
+import schemaEntry from './img/schemaEntry.png';
+import jsonSettings from './img/jsonSettings.png';
+
+# Template-Support
+Für einfacheres Entwickeln von Element-Templates wünschen sich Modellierer und Entwickler in der Regel zwei Dinge:
+
+1. Autocompletion
+2. Syntax-Highlighting
+
+Im Folgenden wird erklärt, wie beides in VS-Code einfach zu erlangen ist.
+
+## Autocomplete
+Folgende Möglichkeiten stehen Ihnen zur Verfügung um Autocompletion für Element-Templates zu erlangen.
+
+### Von Miranum-Console generierte Element-Templates
+Unsere generierten Element-Templates enthalten automatisch ein Autocomplete Support in VS-Code.
+Sie müssen in diesem Fall also nichts machen.
+
+### Eigene Element-Templates
+Durch das Hinzufügen einer "$schema" Property am Kopfe des Templates kann Autocompletion supportet werden.
+Das Schema von Camunda für Element-Templates finden Sie unter: 
+[https://unpkg.com/@camunda/element-templates-json-schema/resources/schema.json](https://unpkg.com/@camunda/element-templates-json-schema/resources/schema.json)
+
+Es sollte danach folgendermaßen ausschauen:
+<img src={singleSchema} alt="singleSchema"/>
+
+### Autocompletion für alle Schemas
+Credit an Bruno Paz und seinen [Artikel](https://dev.to/brpaz/how-to-create-your-own-auto-completion-for-json-and-yaml-files-on-vs-code-with-the-help-of-json-schema-k1i), 
+in dem er durch folgende Schritte erklärt, wie Autocompletion für Visual-Studio-Code zu erlangen ist.
```

Comment:
Grundsätzlich finde ich es eine sehr gute Idee gute/hilfreiche Artikel zu verlinken und den Autor zu würdigen. Ich würde das hier aber etwas professioneller schreiben.

Mit folgenden Schritte können Sie die automatische Vervollständigung in VS-Code einstellen: